server/app.py

```diff
@@ -1,38 +1,45 @@
+import re
 from pathlib import Path
 
 import boto3
 import yaml
-from cryptography.hazmat.primitives import hashes, serialization
-from cryptography.hazmat.primitives.asymmetric import padding
 from flask import Flask, request
 from jose import jwt
 
+from lib.key_helper import PublicKeyManager
+
 app = Flask(__name__)
 
-# create path for the config file
+# Load configuration from the config file
 config_path = Path(__file__).parent / "config.yaml"
-
 with open(config_path, "r") as config_file:
     config = yaml.safe_load(config_file)
 
-# Specify the file name that will appear in the bucket once uploaded
-object_name = "test.txt"
-
 # URL expiration time in seconds
-url_expiration = 3600  # 1 hour
+URL_EXPIRATION = 3600 # 1 hour
```

Comment:
We don't want to make URL_EXPIRATION, TOKEN_EXPIRATION, MAX_FILE_NAME_LENGTH configurations?

Reply:
I agree, eventually we should build this out, but I was of the mind of keeping it simple while it's a prototype. I even debated with myself removing the JWT token expiration to simplify the client config.